browser/brave_profile_prefs.cc

```diff
@@ -311,18 +317,21 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterListPref(kBraveTodayWeeklyCardViewsCount);
   registry->RegisterListPref(kBraveTodayWeeklyCardVisitsCount);
 
-  // Brave Wallet
-#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)
   registry->RegisterIntegerPref(kBraveWalletPrefVersion, 0);
```

Comment:
can we add a prefix for these so we don't confuse them with brave_wallet prefs? Especially since some of these are kBraveWallet...

Reply:
yep but I'll do it in the next PR as discussed on Slack.

Reply:
n/m did it in this PR since CI needed to run anyway